test/cppsuite/test_harness/util/debug_utils.h

```diff
@@ -30,29 +30,71 @@
 #define DEBUG_UTILS_H
 
 #include <iostream>
+#include <sstream>
+#include <thread>
+
+extern "C" {
+#include "wt_internal.h"
+#include "test_util.h"
+}
 
 /* Define helpful functions related to debugging. */
 namespace test_harness {
 
-#define DEBUG_ERROR 0
-#define DEBUG_WARN 1
-#define DEBUG_INFO 2
-#define DEBUG_TRACE 3
+/* Possible log levels */
+#define LOG_ERROR 0
+#define LOG_WARN  1
+#define LOG_INFO  2
+#define LOG_TRACE 3
+
+/* Order of elements in this array corresponds to the definitions abouve. */
+const char * const LOG_LEVELS[] = { "ERROR", "WARN", "INFO", "TRACE" };
+
+/* Current log level */
+static int64_t _trace_level = LOG_TRACE;//LOG_WARN;
+
+/* If true both date and time will be logged, if false only time. */
+static bool _include_date = true;
+
+void get_time(char *time_buf, size_t buf_size) {
+
+    size_t alloc_size;
+    struct timespec tsp;
+    struct tm *tm, _tm;
 
-static int64_t _trace_level = DEBUG_WARN;
+    __wt_epoch(NULL, &tsp);
+    tm = localtime_r(&tsp.tv_sec, &_tm);
+
+    alloc_size = strftime(time_buf, buf_size, 
+      _include_date ? "[%Y-%m-%d][%H:%M:%S" : "[%H:%M:%S", tm);
```

Comment:
Missing closing bracket after `S` ?

Reply:
It will be added after nanoseconds, see `__wt_snprintf` below.